asserts/asserts.go

```diff
@@ -246,12 +235,12 @@ func buildAssertion(headers map[string]string, body, content, signature []byte)
 	}
 
 	if _, err := checkMandatory(headers, "authority-id"); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("assertion %v", err)
```

Comment:
@niemeyer probably better a prefix, these two are the special ones, the other usages are inside the builders themselves get a error prefix already naturally, so they would all need to pass "" for what